nanopub/publication.py

```diff
@@ -32,8 +32,8 @@ def __init__(self, rdf=None, source_uri=None):
 
     @classmethod
     def from_assertion(cls, assertion_rdf, uri=DEFAULT_NANOPUB_URI, introduces_concept=None,
```

Comment:
The code in this method is now a lot clearer (to me)! I'm torn between the simplicity of having ```attribute_to_profile```/```attribute_assertion_to_profile``` be bools, or reducing the number of arguments by having e.g. ```attribute_assertion_to_profile``` default to ```True``` in the case that ```assertion_attributed_to``` remains ```None```. I guess the current way is better because someone might (in principle) not want to attribute the assertion to anyone?

Reply:
Yes, that is why me and Djura chose for this solution. Because indeed someone might not want to attribute assertion to anyone, but more importantly if we attribute to the profile by default someone might accidentally publish something that is attributed to themselves without them knowing!